gitlens/gitlens-plus.md

```diff
@@ -100,16 +108,61 @@ Filter options can be accessed from the filter dropdown. This provides the abili
 
 <img src="/wp-content/uploads/filter-options.png" srcset="/wp-content/uploads/filter-options@2x.png" class="img-bordered img-responsive center">
 
+### Minimap (Experimental)
+
+The Minimap provides an additional dimension to the Commit Graph. You can quickly see the activity of the repository, see the HEAD/upstream, branches (local and remote), and easily jump to them. Select the Toggle Minimap icon in the right corner of the Commit Graph top bar to toggle the Minimap on and off. 
+
+<img src="/wp-content/uploads/gl-minimap.png" class="img-bordered img-responsive center">
+
+Minimap Overview:
+
+- Reads left to right - left is the most recent and the right is older
+- Highlighted Region: Commit Graph area that is in view
+- Green Lines: HEAD and upstream
+- Blue Lines: Tags
```

Comment:
The colors and shapes on these got updates last week. We might want to use an updated image for the screenshot as well.

Most of the indicators are blocks now (with some exceptions like HEAD, upstream, search results). There are two rows of blocks, stacked on top of each other.
Tags are now orange blocks, in the upper block row.
Local branches and remote branches are now blue blocks. Local in the lower block row and remote in the upper block row.
Stashes are pink blocks in the lower block row.
Search results are yellow vertical lines.